src/xrpld/app/tx/detail/Escrow.cpp

```diff
@@ -392,6 +461,13 @@ EscrowFinish::doApply()
     if (!slep)
         return tecNO_TARGET;
 
+    // Order of processing the release conditions (in order of performance):
```

Comment:
EscrowFinish::doApply() became too bid, please divide it

Reply:
Can this be done later?